test/common/upstream/upstream_impl_test.cc

```diff
@@ -706,6 +707,34 @@ TEST(PrioritySet, Extend) {
   }
 }
 
+// Resolve zero hosts, while using health checking.
```

Comment:
nit: this is not really what this test is about. :)

Reply:
Haha, nice catch